roles/common/tasks/main.yml

```diff
@@ -36,13 +32,17 @@
     dest=/home/jenkins/apache-maven-3.2.5-bin.tar.gz
     mode=0640
 
+- file: path=/home/jenkins/apache-maven-3.2.5-bin.tar.gz owner=jenkins group=jenkins mode=0644
```

Comment:
If you want to split it up, you can omit the "mode" attribute from get_url.

But for the sake of simplicity I'd simply give all three, i.e. mode, owner
and group directly for get_url and be done with it.